pep-0708.rst

```diff
@@ -160,10 +160,15 @@ Locations".
 Repository "Tracks" Metadata
 ----------------------------
 
-To enable one repository to extend another, this PEP allows the extending
-repository to declare that it "tracks" another repository by adding the URL
-of the repository that it is extending. This is exposed in JSON as the key
-``meta.tracks`` and in HTML as a meta element named ``pypi:tracks``.
+To enable one repository to host a project that is intended to "extend" a
+project that is hosted at other repositories, this PEP allows the extending
+repository to declare that a particular project "tracks" a project at another
+repository or repositories by adding the URLs of the project and repositories
+that it is extending.
+
+This is exposed in JSON as the key ``meta.tracks`` and in HTML as a meta element
+named ``pypi:tracks``.
```

Comment:
This should note that the key/element needs to be on the *project* page, not on the root ("list of projects") page. In particular for the JSON case, both pages have a "meta" key, so as it stands this is ambiguous.